### DISCUSSION TITLE
Question About Submitting

### DISCUSSION THREAD

**Discussion by wang975679801** [PaddlePaddle/PaddleSpatial#55] [BaiduKDDCup2022] [answered]
We submit the code and get the error:
[OOMKilled] job failed because of Insufficient memory.
How much memroy could we use while making evaluation?

**Comment by ramber1836** [comment, marked as answer]
The max memory is 16 GB right now.

**Reply by wang975679801** [reply]
Is the online test input has the same size as 0001in.csv? We always get the OOM Error.

**Reply by ramber1836** [reply]
For each prediction, the records collected in 14 days are taken as input and 2 days as output, which is the same as the toy dataset.

**Reply by luckyyangrun** [reply]
Call for more memory !!!!

**Reply by HHHLLL1** [reply]
On my local machine(16G), using the evaluation code in apps/wpf_baseline_gru/kddcup22-sdwpf-evaluation/paddlepaddle is ok, but get OOM when using code in apps/wpf_baseline_gru.

**Comment by ramber1836** [comment]
Have you tested your code on your local machine? How about the resources your model costs? 
It would be helpful if you can show us more details of resource occupation when you evaluate your model, including CPU usage, Memory Usage, etc. 
@wang975679801 @luckyyangrun

**Comment by ramber1836** [comment]
The latest version of evaluation code is the script located in
apps/wpf_baseline_gru/kddcup22--sdwpf-evaluation/paddlepaddle .
The online test also employs the latest version of the evaluation script.
